<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisconnectButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Producer_txtBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>211</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> IP</string>
    </property>
   </widget>
   <widget class="QSlider" name="HS_MAX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_MIN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_Timing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD_MAX">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD_MIN">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing(s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Timing">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>HS_Timing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LBL_Timing</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HS_MAX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LCD_MAX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HS_MIN</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LCD_MIN</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
